.github/workflows/pre-commit.yml

```diff
@@ -1,12 +1,12 @@
-name: Lint
+name: pre-commit
```

Comment:
It makes more sense to me, but I'm OK with renaming back, since `lint.yml` is the pattern:

https://github.com/search?q=org%3Apython%20action-pre-commit-uv&type=code